packages/frameworks/src/frameworks.ts

```diff
@@ -17,6 +17,73 @@ const { readdir, readFile, unlink } = promises;
  */
 
 export const frameworks = [
+  {
+    name: 'Remix',
+    slug: 'remix',
+    demo: 'https://remix.examples.vercel.com',
+    logo: 'https://raw.githubusercontent.com/vercel/vercel/main/packages/frameworks/logos/remix.svg',
+    tagline: 'Build Better Websites',
+    description: 'A new Remix app — the result of running `npx create-remix`.',
+    website: 'https://remix.run',
+    detectors: {
+      every: [
+        {
+          path: 'package.json',
+          matchContent:
+            '"(dev)?(d|D)ependencies":\\s*{[^}]*"remix":\\s*".+?"[^}]*}',
+        },
+      ],
+    },
+    settings: {
+      installCommand: {
+        placeholder: '`yarn install` or `npm install`',
+      },
+      buildCommand: {
+        value: 'remix build',
+        placeholder: '`npm run build` or `remix build`',
+      },
+      devCommand: {
+        value: 'remix dev',
+        placeholder: 'remix dev',
+      },
+      outputDirectory: {
+        value: 'public',
+      },
+    },
+    dependency: 'remix',
+    getFsOutputDir: async () => 'public',
+    getOutputDirName: async () => 'public',
+    defaultRoutes: [
+      {
+        src: '^/build/(.*)$',
+        headers: { 'cache-control': 'public, max-age=31536000, immutable' },
+        continue: true,
+      },
+      {
+        handle: 'filesystem',
+      },
+      {
+        src: '/(.*)',
+        dest: '/api',
```

Comment:
This doesn't seem right. Is there always a `/api/index.js` file for every remix app?

Reply:
Yup! Remix is designed edge-first, so they have a rendering fallback for all routes.

On Vercel, they're gonna be Lambdas for now, until we expose Edge Functions as a user-facing primitive (I discussed this with them on a call earlier).